pyproject.toml

```diff
@@ -13,7 +13,7 @@ classifiers = [
 description = "Ophyd devices and other utils that could be used across DLS beamlines"
 dependencies = [
     "ophyd",
-    "ophyd-async>=0.3a2",
+    "ophyd-async @ git+https://github.com/bluesky/ophyd-async.git@c17a372d9d3389bad030673f333e04f1e8659b0c", # Needed for https://github.com/bluesky/ophyd-async/pull/273, unpin when merged
```

Comment:
I guess you mean "when included in a release" as opposed to "when merged", since it's already merged?

Reply:
Could we just do a new alpha release today @callumforrester or @abbiemery?

Reply:
We shouldn't do a release of `ophyd_async` until https://github.com/DiamondLightSource/dodal/issues/492 is sorted

Reply:
Oh god we're getting into a horrible web of everything depending on everything else again...

As a side note, the `i03` system tests don't fail for me on either `main` or this branch :shrug: . But it looks like it should be a valid fix...

Reply:
It's perfectly possible I'm just going mad with my dev environment, trying to juggle about 4 different issues at the same time today